kitsune/wiki/urls.py

```diff
@@ -161,3 +161,14 @@
         r"^/discuss/watch_locale$", kbforums_views.watch_locale, name="wiki.discuss.watch_locale"
     ),
 ]
+
+urlpatterns += [
+    # Redirect for pocket articles
+    # This assumes pocket redirects take the form of:
+    # /kb/pocket/article/<article_id>-<document_slug>
+    re_path(
```

Comment:
Pocket will have the same URL structure for the KB similar to the other products. For example: 
Pocket article `https://help.getpocket.com/article/1142-firefox-new-tab-recommendations` will redirect to `https://support.mozilla.org/en-US/kb/irefox-new-tab-recommendations`

Basically we are keeping everything that follows after the `ID-` and we are considering the slug. If we don't have such an article we are redirecting to the product landing page. I suspect that will be `https://support.mozilla.org/en-US/products/pocket`

Reply:
We need an additional piece in the URL for incoming pocket redirects (aka the /pocket/) to give us the ability to know these were incoming pocket articles vs. any other type of article request.  So if we set up the redirect from pocket to land at just `/kb/article-slug` we are going to have to know in some other way where the request came from so we don't just fail **all** bad kb article requests to `/products/pocket` . I think having the `/pocket/` piece in there (doesn't have to be called that, or in that order, etc.) is easier than trying to catch the redirect sources in Django. 
I guess alternately we can get a redirect to `/kb/my-article-from-pocket?pocket=true' or something like that. The slug or data passed doesn't really matter to me, but I don't think there is another way to know for sure that the visitor tried to go to `help.getpocket.com` first.
Hope I understood what you were after here - let me know!